sixpack/sixpack.py

```diff
@@ -106,10 +106,12 @@ def get_response(self, endpoint=None, params=None):
         try:
             response = requests.get(url, params=params, timeout=self.timeout)
             if response.status_code != 200:
-                ret = "{{\"status\": \"failed\", \"response\": {0}}}".format(response.content)
+                try:
+                    content = response.json()
+                except Exception:
```

Comment:
I don’t *know* what exceptions this may throw — as you know, Python isn’t Java and isn’t big on specifying exceptions that may be thrown somewhere.

The point of this try/except block is to provide at least some error response content if decoding the response as JSON fails *for whatever reason*. `except Exception` is an appropriate expression of “whatever reason” in this situation, because it a) doesn’t loose any information (it will be unformatted, but it will be there) and b) doesn’t risk unexpectedly propagating some unexpected (by me) exception out.